sdk/keyvault/azure-security-keyvault-administration/swagger/autorest.md

```diff
@@ -52,14 +52,14 @@ context-client-method-parameter: true
 These settings apply only when `--tag=rbac` is specified on the command line.
 
 ``` yaml $(tag) == 'rbac'
-input-file: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/rbac.json
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.3/rbac.json
 title: KeyVaultAccessControlClient
 ```
 
 ### Tag: backuprestore
 These settings apply only when `--tag=backuprestore` is specified on the command line.
 
 ``` yaml $(tag) == 'backuprestore'
-input-file: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/backuprestore.json
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.3/backuprestore.json
```

Comment:
I'd recommend using a specific commit here, ideally `e2ef44b87405b412403ccb005bfb3975411adf60` (same for any other autorest files)

Reply:
```suggestion
input-file: https://github.com/Azure/azure-rest-api-specs/blob/e2ef44b87405b412403ccb005bfb3975411adf60/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.3/backuprestore.json
```